x-pack/solutions/observability/plugins/profiling/public/components/differential_topn_functions_grid/index.tsx

```diff
@@ -226,109 +238,114 @@ export function DifferentialTopNFunctionsGrid({
   const rowCount = Math.max(sortedBaseRows.length, sortedComparisonRows.length);
 
   return (
-    <>
-      <EuiDataGrid
-        data-test-subj="profilingDiffTopNFunctionsGrid"
-        css={css`
-          .thickBorderLeft {
-            border-left: ${theme.euiTheme.border.thick} !important;
-          }
-        `}
-        aria-label={i18n.translate(
-          'xpack.profiling.onWeelkDiffTopN.euiDataGrid.topNFunctionsLabel',
-          {
-            defaultMessage: 'TopN functions',
-          }
-        )}
-        columns={columns}
-        leadingControlColumns={[
-          {
-            id: 'actions',
-            width: 40,
-            headerCellRender() {
-              return (
-                <EuiScreenReaderOnly>
-                  <span>
-                    {i18n.translate('xpack.profiling.topNFunctionsGrid.span.controlsLabel', {
-                      defaultMessage: 'Controls',
-                    })}
-                  </span>
-                </EuiScreenReaderOnly>
-              );
-            },
-            rowCellRender: function RowCellRender({ rowIndex }) {
-              function handleOnClick() {
-                const row = sortedBaseRows[rowIndex];
-                const currentFrameId = getFrameIdentification(row.frame);
-                const compareRow = sortedComparisonRows.find(
-                  (item) => getFrameIdentification(item.frame) === currentFrameId
+    <EuiFlexGroup direction="column" gutterSize="s">
+      <EuiFlexItem grow={false}>
+        <SearchFunctionsInput onChange={setSearchQuery} />
+      </EuiFlexItem>
+      <EuiFlexItem>
+        <EuiDataGrid
+          data-test-subj="profilingDiffTopNFunctionsGrid"
+          css={css`
+            .thickBorderLeft {
+              border-left: ${theme.euiTheme.border.thick} !important;
```

Comment:
Nit: Can we remove the `!important` here? It looks the same without it from what I see 🤔